fuel-merkle/src/sparse/merkle_tree/node.rs

```diff
@@ -1,5 +1,7 @@
 use crate::{
     common::{
+        Bytes32,
+        Prefix,
```

Comment:
Maybe we could just use some rule in the `.rustfmt.toml` to avoid this change in all files?=D